{"name": "Тест: запрос с несуществующим заказом возвращает ошибку", "status": "passed", "start": 1717930441725, "stop": 1717930442017, "uuid": "b99dc1d8-82e8-4038-ac28-15910b6b7c1f", "historyId": "4df8ca3f1fcfc736c88db5a30124b814", "testCaseId": "4df8ca3f1fcfc736c88db5a30124b814", "fullName": "tests.test_get_order.TestGetOrder#test_get_order_with_non_existent_number_order_code_400", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestGetOrder"}, {"name": "host", "value": "DESKTOP-USF6L0H"}, {"name": "thread", "value": "16456-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}